src/style.css

```diff
@@ -1,3 +1,74 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto/Roboto-Medium.ttf);
+}
+
 body {
   margin: 0;
 }
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: block;
+  float: right;
+}
+
+li a {
+  text-align: center;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  flex-direction: row-reverse;
+}
+
+.link-8,
+.link-5 {
+  color: #00acdc;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  font-family: Roboto, sans-serif;
+  height: 60px;
+  display: flex;
+  font-size: 12px;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  padding: 0 20px;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.link-8::after {
+  content: '';
+  display: block;
+  position: relative;
+  background-color: #00acdc;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+}
+
+img {
+  margin-left: 50px;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header?tab=readme-ov-file#checklist): CSS Variable is used for a blue color